lib/matplotlib/tests/test_contour.py

```diff
@@ -233,6 +234,32 @@ def test_labels(split_collections):
     _maybe_split_collections(split_collections)
 
 
+def test_label_contour_start():
+    # Set up data and rcparams that result in automatic labelling adding the
+    # label to the start of a contour
+    plt.rcdefaults()
```

Comment:
I suspect this is not robust to future version changes, but so far I have not found a combination of options that get the test to work without it.

Reply:
Then I went back and read @greglucas’s comment again…